docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss

```diff
@@ -126,3 +126,24 @@
     }
   }
 }
+
+@mixin hb-blockquote {
+  @include hb-body--medium;
+  color: darken($su-color-driftwood, 45%);
+  font-style: italic;
+  line-height: 140%;
+  position: relative;
+  quotes: '"' '"';
```

Comment:
I didn't know this was a thing!

![giphy](https://user-images.githubusercontent.com/2486846/75206510-9bb5d880-5744-11ea-9bc4-c2a40bf0870a.gif)